frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -128,13 +130,38 @@ class SchemaItem extends React.Component<{
         const { rows } = this.state;
         let copiedRows = [...rows];
         // replace new line with space, remove single quotes since we use that to quote fields with space when saving
-        copiedRows.splice(idx, 1, {
-            ...rows[idx],
-            [key]: e.target.value.replace(/\n/g, ' ').replace("'", ''),
-        });
+        copiedRows.splice(
+            idx,
+            1,
+            this.autofillTypes({
+                ...rows[idx],
+                [key]: e.target.value.replace(/\n/g, ' ').replace("'", ''),
+            }),
+        );
         this.setState({ rows: [...copiedRows] });
     };
 
+    autofillTypes = (row) => {
+        // Do not fill if how to render is defined.
+        if (row['post-processor'] !== null) {
+            return row;
+        }
+
+        switch (row['generalized-path'].toLowerCase()) {
```

Comment:
Can we instead use a dictionary with a name like `DEFAULT_POST_PROCESSOR` so it's easier to configure in the future?

Reply:
Thanks, I added it to constants.